src/natcap/invest/finfish_aquaculture/finfish_aquaculture.py

```diff
@@ -37,8 +37,11 @@
                 "tables). Additionally, the numbers underneath this "
                 "farm identifier name must be unique integers for all "
                 "the inputs."),
-            "type": "freestyle_string",
+            "type": "option_string",
```

Comment:
dynamically populated dropdowns have had the `freestyle_string` type which is kinda incorrect but worked. It doesn't work any more because the `type` property determines what kind of input field is rendered. I don't really like having the set of options here in `validation_options.options`, but it doesn't really make sense to go in the UI spec either

Reply:
oof glad you caught that!